### PR TITLE
1649 homepage

### LINKED ISSUE #1567: Update Landscape logo in homepage takeunder
## Summary

The Landscape logo used in the current homepage takeover is out of date.

## Process

Go to https://www.ubuntu.com/ and scroll down to the takeunder section

## Current and expected result

It should use this logo instead: https://assets.ubuntu.com/v1/4964d639-landscape-logo.svg

### LINKED ISSUE #1649: Implement new homepage in VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* converted homepage to vbt and the new design
* updated Landscape logo

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/))
- compare to the [copy doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit) and the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/Homepage/Deliverables?preview=homepage+v5+mix.png)

## Issue / Card

Fixes #1649 
Fixes #1567

### CODE REVIEW

**Comment by yaili** [comment]
@pmahnke @nottrobin demo doesn't work

**Comment by nottrobin** [comment]
@yaili yeah, demos are poorly. I'll see if I can nurse it back to health tomorrow, if @WillMoggridge isn't back.

File: static/sass/section/_v1_homepage.scss [hunk 1]
```diff
@@ -0,0 +1,249 @@
+// homepage patterns
+.homepage {
+  .p-ubuntu-intro {
+
+    @media only screen and (min-width : 769px) {
+      clear: both;
+      display: -webkit-box;
+      display: -moz-box;
+      display: -ms-flexbox;
```
Comment (barrymcgee) [inline static/sass/section/_v1_homepage.scss]: We have autoprefixer in the build chain, right?

File: static/sass/section/_v1_homepage.scss [hunk 2]
```diff
@@ -0,0 +1,249 @@
+// homepage patterns
+.homepage {
+  .p-ubuntu-intro {
+
+    @media only screen and (min-width : 769px) {
```
Comment (barrymcgee) [inline static/sass/section/_v1_homepage.scss]: Use `$breakpoint-medium + 1`

File: static/sass/section/_v1_homepage.scss [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+      clear: both;
+      display: -webkit-box;
+      display: -moz-box;
+      display: -ms-flexbox;
+      display: flex;
+      margin-bottom: 10px;
```
Comment (barrymcgee) [inline static/sass/section/_v1_homepage.scss]: Use Vanilla spacing variables

File: static/sass/section/_v1_homepage.scss [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+        }
+      }
+    }
+  }
+
+  /// Featured row, for takeunders
+  .p-takeunder {
+    border-radius: 2px;
+    color: $color-x-light;
+    padding: 2rem;
```
Comment (barrymcgee) [inline static/sass/section/_v1_homepage.scss]: Vanilla spacing variables

File: static/sass/styles-v1.scss [hunk 5]
```diff
@@ -199,6 +200,18 @@ fieldset {
   text-transform: uppercase;
 }
 
+.p-heading--insights__title {
+  color: $color-dark;
+  font-size: 1rem;
```
Comment (barrymcgee) [inline static/sass/styles-v1.scss]: spacing variables

**Comment by pmahnke** [comment]
@yaili demo - http://www.ubuntu.com-1649-homepage.demo.haus/

**Comment by barrymcgee** [comment]
screen shot 2017-06-06 at 11 53 58

**Comment by barrymcgee** [comment]
@yaili Demo: http://www.ubuntu.com-1649-homepage.demo.haus/

**Comment by pmahnke** [comment]
@barrymcgee updated where I could... or moved to rems
not sure what the QA issue was... guessing add is-bordered above the juju image?

**Comment by barrymcgee** [comment]
@pmahnke QA issue is huge space between content and image on the left, zoom out your screen to see how it looks on large screens..

**Comment by yaili** [comment]
Design review:
- [ ] In the icons row, can we remove the dotted lines outside of the icons, and only keep the lines in between icons?
- [ ] This isn't in the new design, but were we going to add the labels of the icons always visible instead of just on hover?
- [ ] Add border above the icons row
- [ ] The dotted lines in the icons row should align centre vertical with the icons, right now they are aligning top
- [ ] The image in the containers row should align left with the text blocks in IoT and Desktop
- [ ] When the viewport is a little bit smaller (but still at L size) the takeunder cards start to not align left with the text block above - can this be fixed?
- [ ] The position of the cloud row image needs to be improved, currently at smaller L sizes it breaks (screenshot at 910px) - if the white space on the left of the image is cropped it might be easier to fix
screen shot 2017-06-06 at 13 55 52

- [ ] There shouldn't be a white 1px line between the takeover and the navigation at S screen
- [ ] At S screen, the takeover image shouldn't touch the button
- [ ] Is it OK that the homepage has almost no images at all at S screen (just the icons)? I think @frankiegranato should confirm if this is acceptable

The design had the paragraphs in each of the sections as 21px font size, but I think 16px works.

**Comment by yaili** [comment]
- [ ] Can we u-align--center the takeunder images?
- [ ] Can the takeunder images be in col-4 and the text blocks in col-8?
- [ ] I think the text blocks in cloud and containers are a bit too much to the right, here is the design with the grid over it, they should align left, and be closer to the image
screen shot 2017-06-07 at 11 35 46

- [ ] Some issues mentioned above haven't been done yet (the unticked ones)

**Comment by pmahnke** [comment]
@yaili updated

**Pull request merged**